apps/fabric-website/src/pages/Controls/ControlsAreaPage.tsx

```diff
@@ -6,16 +6,21 @@ import { IPageJson } from 'office-ui-fabric-react/lib/common/DocPage.types';
 
 export interface IControlsPageProps extends IPageProps<Platforms> {}
 
-const apiRequireContext = require.context('@uifabric/api-docs/lib/pages/office-ui-fabric-react');
+const apiRequireContext = [
+  require.context('@uifabric/api-docs/lib/pages/office-ui-fabric-react'),
```

Comment:
Oh actually, I have another idea which will scale better: go back to a single context and use negative lookahead! (I confirmed it works in all browsers.) 
`require.context('@uifabric/api-docs/lib/pages/', true, /^(?!references).*/)`

require.context reference: https://webpack.js.org/api/module-methods/#requirecontext